src/test/java/com/iexec/worker/compute/post/web2/UploaderServiceTests.java

```diff
@@ -22,32 +22,32 @@
 import com.dropbox.core.v2.users.DbxUserUsersRequests;
 import com.dropbox.core.v2.users.FullAccount;
 import com.iexec.common.result.ComputedFile;
+import com.iexec.common.result.ResultModel;
+import com.iexec.common.utils.FeignBuilder;
+import com.iexec.worker.api.IexecProxyClient;
 import com.iexec.worker.compute.post.PostComputeException;
-import org.junit.jupiter.api.BeforeEach;
+import feign.Logger;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
 import org.junit.jupiter.api.io.TempDir;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.NullSource;
 import org.junit.jupiter.params.provider.ValueSource;
-import org.mockito.InjectMocks;
-import org.mockito.Mock;
-import org.mockito.MockitoAnnotations;
-import org.mockito.Spy;
-import org.springframework.http.ResponseEntity;
-import org.springframework.web.client.RestTemplate;
+import org.mockito.*;
```

Comment:
This global import with `*` should be avoidable here as well.

Reply:
Resolve in [f700da3](https://github.com/iExecBlockchainComputing/tee-worker-post-compute/pull/120/commits/f700da35df8ce418a03fa0901abe532276178cb1)